### PR TITLE
Fixes broken links in Core docs v2 - Closes #1232

### LINKED ISSUE #1232: Fix broken links in Core docs v2.1.7
Update and fix the following broken links in the pages listed below:

lisk-core/v2/index.html 
lisk-core/v2/setup/application.html
lisk-core/v2/setup/docker.html 
lisk-core/v2/setup/source.html 
lisk-core/v2/management/commander.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1232

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -6,8 +6,9 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-aliases: troubleshooting.adoc, getting-started.adoc
 :page-no-previous: true
 :page-next: /lisk-core/interact-with-the-api.html
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:page-next: /lisk-core/v2/interact-with-the-api.html
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -6,8 +6,9 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-:page-next-title: Interact with the API
+:page-next-title: Interact with the network
 :v_sdk: master
+:v_core: v2
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: remove this

File: modules/ROOT/pages/management/commander.adoc [hunk 3]
```diff
@@ -3,10 +3,11 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: A reference of all Lisk Commander application commands to manage Lisk Core via the command-line.
 :page-aliases: administration/commander.adoc
 :toc:
-:page-next: /lisk-core/management/configuration.html
+:v_core: v2
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]: remove

File: modules/ROOT/pages/management/commander.adoc [hunk 4]
```diff
@@ -3,10 +3,11 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+:page-next: /management/configuration.html
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
:page-next: /lisk-core/v2/management/configuration.html
```

File: modules/ROOT/pages/management/commander.adoc [hunk 5]
```diff
@@ -3,10 +3,11 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :page-previous: /lisk-core/setup/commander.html
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
:page-previous: /lisk-core/v2/setup/commander.html
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Not sure, why are you adding the v_core attribute, but never using it? If it is not used, it should be removed. Also, the links are not fixed correctly. Main issue seems to be the `v2/` is missing in the links. I added it now for a few, please correct the rest yourself. Please also check for all other next /  previous links for v2, they actually might have the same problem, even if not broken.

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: OK, will check...Many thanks Mona 👍

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Done 👍

File: modules/ROOT/pages/management/commander.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/management/commander.adoc]: Done 👍

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Hey Mona! Ref, your comments above...Yes, you are correct, I found some more broken links with missing instances of 'v2' so have also fixed them as well 👍

File: modules/ROOT/pages/setup/application.adoc [hunk 6]
```diff
@@ -4,8 +4,11 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-aliases: setup/binary.adoc
 :toc:
 :v_sdk: master
-:page-next: /lisk-core/management/application.html
+
+:page-next: /lisk-core/v2/management/application.html
+:page-previous-title: Interact with the network
 :page-next-title: Application commands
+:page-previous: /lisk-core/interact-with-the-api.html
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/application.adoc]:
```suggestion
:page-previous: /lisk-core/v2/interact-with-the-api.html
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 7]
```diff
@@ -2,8 +2,8 @@
 Mona Bärenfänger <mona@lightcurve.io>
 :description: Describes all requirements and dependencies to setup a Lisk Core node with Lisk Commander.
 :toc:
-:v_sdk: v2
-:v_core: 2.1.7
+:v_sdk: v3
+:v_core: v2
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]:
```suggestion
:v_core: 2.1.7
```

File: modules/ROOT/pages/update/application.adoc [hunk 8]
```diff
@@ -6,7 +6,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :experimental:
 :v_sdk: master
 :page-next: /lisk-core/monitoring.html
```
Comment (Tschakki) [inline modules/ROOT/pages/update/application.adoc]:
```suggestion
:page-next: /lisk-core/v2/monitoring.html
```

File: modules/ROOT/pages/update/commander.adoc [hunk 9]
```diff
@@ -3,10 +3,10 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: How to upgrade Lisk Core to the latest version via Lisk Commander.
 :page-aliases: upgrade/commander.adoc
 :toc:
-:v_core: 2.1.7
-:page-next: /lisk-core/monitoring.html
-:page-previous: /lisk-core/management/commander.html
-:page-next-title: Monitoring
+
+:page-next: /lisk-core/docker.html
```
Comment (Tschakki) [inline modules/ROOT/pages/update/commander.adoc]:
```suggestion
:page-next: /lisk-core/v2/monitoring.html
```

File: modules/ROOT/pages/update/commander.adoc [hunk 10]
```diff
@@ -3,10 +3,10 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+:page-previous: /lisk-core/v2/management/commander.html
+:page-next-title: Docker image
```
Comment (Tschakki) [inline modules/ROOT/pages/update/commander.adoc]:
```suggestion
:page-next-title: Monitoring
```

File: modules/ROOT/pages/update/docker.adoc [hunk 11]
```diff
@@ -4,7 +4,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-aliases: upgrade/docker.adoc
 :toc:
 :page-next: /lisk-core/monitoring.html
```
Comment (Tschakki) [inline modules/ROOT/pages/update/docker.adoc]:
```suggestion
:page-next: /lisk-core/v2/monitoring.html
```

File: modules/ROOT/pages/update/source.adoc [hunk 12]
```diff
@@ -4,7 +4,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-aliases: upgrade/source.adoc
 :toc:
 :page-next: /lisk-core/monitoring.html
```
Comment (Tschakki) [inline modules/ROOT/pages/update/source.adoc]:
```suggestion
:page-next: /lisk-core/v2/monitoring.html
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/update/source.adoc (hunk 12 above)
Comment (chris529) [reply modules/ROOT/pages/update/source.adoc]: Cool, thanks, Mona, so there were even more!

**Review by Tschakki — APPROVED** [review]
Thanks, looking good now!

**Pull request merged**